docs/docs-beta/docs/reference/workspace-yaml.md

```diff
@@ -0,0 +1,101 @@
+# Workspace File Reference
+
+:::info
+    This reference is only applicable to Dagster OSS, for Dagster Cloud see [the Dagster Cloud Code Locations guide](/dagster-plus/deployment/code-locations)
+:::
+
+The `workspace.yaml` file is used to configure code locations in Dagster. It tells Dagster where to find your code and how to load it.
```

Comment:
Should we call out where this file is stored? (e.g. `$DAGSTER_HOME`)?

Reply:
good idea